.github/workflows/slack-noti.yml

```
@@ -0,0 +1,16 @@
+name: Slack
+on: [pull_request]
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+    name: Slack Notification
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Fire Notification
+        uses: Lubycon/github-reviewer-slack-noti-action@v1.1.2
+        with:
+          slack-bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
```

```suggestion
          slack-bot-token: ${{ secrets.LUBYCON_SLACK_BOT_TOKEN }}
```
으로 바꿔주시고 PR 닫았다가 다시 create 해보시면 잘 될 듯 합니다!